.azure-pipelines/impacted_area_testing/calculate_instance_number.py

```diff
@@ -131,6 +134,8 @@ def main(scripts, topology, branch):
     parser.add_argument("--branch", help="Test branch", type=str, default="")
     args = parser.parse_args()
 
+    logging.basicConfig(level=logging.INFO)
```

Comment:
suggest to add the basic config below the import

Reply:
done